db/migrate/20231005144432_create_organization_profiles.rb

```diff
@@ -0,0 +1,17 @@
+class CreateOrganizationProfiles < ActiveRecord::Migration[7.0]
+  def up
+    create_table :organization_profiles do |t|
+      t.string :email
+      t.string :phone_number
+
+      t.references :location, index: true
+      t.references :organization
```

Comment:
Let's add `null: false` and `foreign_key: true` to these columns for extra robustness 🛡️

Don’t forget to run `rails db:rollback` and `rails db:migrate` (or `rails db:reset` if needed) to refresh the schema.rb after tweaking!